src/ro/siit/LocalStore/Store.java

```diff
@@ -0,0 +1,158 @@
+package ro.siit.LocalStore;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Scanner;
+
+import static ro.siit.LocalStore.Main.productList;
+import static ro.siit.LocalStore.SoldItem.soldList;
+import static ro.siit.LocalStore.StockItem.stockList;
```

Comment:
it's not good to do everything with static... Why does main hold the productList? 
Why doesn't the Store have the soldList and stockList as attributes?